http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/RuntimeRoutesSpec.scala

```diff
@@ -106,7 +106,7 @@ class RuntimeRoutesSpec extends AnyFlatSpec with Matchers with LeonardoTestSuite
       json <- io.circe.parser.parse(inputString)
       r <- json.as[RuntimeConfigRequest.DataprocConfig]
     } yield r
-    decodeResult.leftMap(_.getMessage) shouldBe Left("Minimum required masterDiskSize is 50GB")
+    decodeResult.leftMap(_.getMessage) shouldBe Left(DecodingFailure("Minimum required masterDiskSize is 50GB"))
```

Comment:
Seems like the contract testing fails here?
https://github.com/DataBiosphere/leonardo/actions/runs/4599116548/jobs/8124084892?pr=3252

Reply:
It's not the tests failing; this wasn't compiling. My mistake - fixed